less/core/notify.less

```
@@ -106,3 +105,33 @@
     .transition(background-color @duration ease-in, color @duration ease-in;);
   }
 }
+
+
+.notify:not(.disable-animation) > .notify__popup {
+  .transition(margin-top var(--adapt-notify-duration) @animation-easing, top var(--adapt-notify-duration) @animation-easing, transform var(--adapt-notify-duration) @animation-easing, opacity var(--adapt-notify-duration) @animation-easing, height var(--adapt-notify-duration) @animation-easing;);
```

```suggestion
  .transition(top var(--adapt-notify-duration) @animation-easing, transform var(--adapt-notify-duration) @animation-easing, opacity var(--adapt-notify-duration) @animation-easing, height var(--adapt-notify-duration) @animation-easing;);
```
I can't see margin-top used anywhere.

---

Agreed. Previously [resizeNotify](https://github.com/adaptlearning/adapt-contrib-core/blob/a1b6c57f8608d0a38ef6918ce3bc9bad6961f580/js/views/notifyPopupView.js#L122) set `margin-top` so this has probably carried over from the previous Vanilla [notify.less](https://github.com/adaptlearning/adapt-contrib-vanilla/blob/3c5d65489f989c568a4a8e816788d9804e8cf67f/less/core/notify.less#L5).